Jenkinsfile

```diff
@@ -51,19 +51,30 @@ pipeline {
                     }
                     releaseVersion = pomVersion.replace("-SNAPSHOT","")
                 }
-                // tests are run in prior tests
-                sh "mvn --batch-mode ${dryRun} -Dtag=${pomArtifactId}-${releaseVersion} release:prepare"
-                sh "mvn --batch-mode ${dryRun} release:perform"
+                // I want to retain this commented credential for documentation of how-to use sshagent in case we need it
+                // this credential name must be defined in jenkins credentials config
+                //sshagent(credentials : ['jenkins_git']) {
+                //sshagent(credentials : ['a19251a9-ab43-4dd0-bd76-5b6dba9cd793']) {
+                    // tests are run in prior tests
+                    sh "mvn --batch-mode ${dryRun} -Dtag=${pomArtifactId}-${releaseVersion} release:prepare"
```

Comment:
So you're able to push changes to git without the SSH key?

Reply:
not really. It uses the default key assigned to the jenkins user in the ssh/config

I left this in as a comment even though it worked as a documentation of how it works.

Do you think I should just leave it in since it works instead of commenting it?

Reply:
I think leaving it commented is fine.